src/editor/bin/Oni2.re

```diff
@@ -15,6 +15,12 @@ open Oni_Neovim;
 
 module Core = Oni_Core;
 
+/**
+   This allows a stack trace to be printed when exceptions occur
+   TODO: trigger this only if oni is in DEBUG mode
+ */
+Printexc.record_backtrace(true) |> ignore;
```

Comment:
This will hurt performance - can we find a way to conditionally enable this? How about we check an `ONI_DEBUG` environment variable?

Reply:
sounds good didn't realise there'd be a performance hit, was definitely intending that to only run under certain circumstances